lib/node_modules/@stdlib/stats/base/ndarray/dnanmeanwd/README.md

```diff
@@ -0,0 +1,153 @@
+
+> Compute the [arithmetic mean][arithmetic-mean] of a one-dimensional double-precision floating-point ndarray, ignoring `NaN` values and using Welford's algorithm.
+
+<section class="intro">
+
+The [arithmetic mean][arithmetic-mean] is defined as
+
+<!-- <equation class="equation" label="eq:arithmetic_mean" align="center" raw="\mu = \frac{1}{n} \sum_{i=0}^{n-1} x_i" alt="Equation for the arithmetic mean."> -->
+
+```math
+\mu = \frac{1}{n} \sum_{i=0}^{n-1} x_i
+```
+
+<!-- <div class="equation" align="center" data-raw-text="\mu = \frac{1}{n} \sum_{i=0}^{n-1} x_i" data-equation="eq:arithmetic_mean">
+    <img src="https://cdn.jsdelivr.net/gh/stdlib-js/stdlib@42d8f64d805113ab899c79c7c39d6c6bac7fe25c/lib/node_modules/@stdlib/stats/base/ndarray/dnanmeanwd/docs/img/equation_arithmetic_mean.svg" alt="Equation for the arithmetic mean.">
+    <br>
+</div> -->
+
+<!-- </equation> -->
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var dnanmeanwd = require( '@stdlib/stats/base/ndarray/dnanmeanwd' );
+```
+
+#### dnanmeanwd( arrays )
+
+Computes the [arithmetic mean][arithmetic-mean] of a one-dimensional double-precision floating-point ndarray, ignoring `NaN` values and using Welford's algorithm.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+var ndarray = require( '@stdlib/ndarray/base/ctor' );
+
+var xbuf = new Float64Array( [ 1.0, -2.0, NaN, 2.0 ] );
+var x = new ndarray( 'float64', xbuf, [ 4 ], [ 1 ], 0, 'row-major' );
+
+var v = dnanmeanwd( [ x ] );
+// returns ~0.333
+```
+
+The function has the following parameters:
+
+-   **arrays**: array-like object containing a one-dimensional input ndarray.
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   If provided an empty one-dimensional ndarray, the function returns `NaN`.
+
+</section>
+
+<!-- /.notes -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var uniform = require( '@stdlib/random/base/uniform' );
+var filledarrayBy = require( '@stdlib/array/filled-by' );
+var bernoulli = require( '@stdlib/random/base/bernoulli' );
+var ndarray = require( '@stdlib/ndarray/base/ctor' );
+var ndarray2array = require( '@stdlib/ndarray/to-array' );
+var dnanmeanwd = require( '@stdlib/stats/base/ndarray/dnanmeanwd' );
+
+function rand() {
+    if ( bernoulli( 0.8 ) < 1 ) {
+        return NaN;
+    }
+    return uniform( -50.0, 50.0 );
+}
+
+var xbuf = filledarrayBy( 10, 'float64', rand );
+var x = new ndarray( 'float64', xbuf, [ xbuf.length ], [ 1 ], 0, 'row-major' );
+console.log( ndarray2array( x ) );
+
+var v = dnanmeanwd( [ x ] );
+console.log( v );
+```
+
+</section>
+
+<!-- /.examples -->
+
+* * *
+
+<section class="references">
+
+## References
+
+-   Welford, B. P. 1962. "Note on a Method for Calculating Corrected Sums of Squares and Products." _Technometrics_ 4 (3). Taylor & Francis: 419–20. doi:[10.1080/00401706.1962.10490022][@welford:1962a].
```

Comment:
Ensure you are using the correct citation identifier. In this case, I corrected the identifier to `@welford:1962a`.